future-apps/agent/contracts/standards/ERC1271.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity 0.4.24;
 // Rationale: https://github.com/ethereum/EIPs/issues/1271#issuecomment-462719728
 
 contract ERC1271 {
-    bytes4 constant public ERC1271_INTERFACE_ID = this.isValidSignature.selector;
+    bytes4 constant public ERC1271_INTERFACE_ID = 0xfb855dc9; // this.isValidSignature.selector
```

Comment:
Note that this was giving a warning; I guess solidity doesn't know this is known at compile-time